unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/SphereShape/Handler/ECSSphereShapeHandler.cs

```diff
@@ -30,9 +30,12 @@ public void OnComponentRemoved(IParcelScene scene, IDCLEntity entity)
 
         public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBSphereShape model)
         {
+            if (lastModel != null && lastModel.Equals(model))
```

Comment:
I don't think so. If the model is exactly the same the component will be already behaving the way you want. This is done because if from the code you use `getMutableGroupOf ` all the components will be updated even if you don't change them